docs/UserGuide.md

```diff
@@ -344,11 +344,13 @@ Format: `COMMAND help`
 
 * If `COMMAND` is not present,
     * list the available commands and the link to the User Guide.
-![result for 'help'](images/helpAllMockup.png)
+    
+<img src="images/helpAllMockup.png" alt="result for 'help'" width="400px">
 
 * If `COMMAND` is present
     * list the command description and example usage.
-![result for 'help command'](images/helpPerCommandMockup.png)
+    
```

Comment:
the lack of new line before the image is intentional. Will it mess with the width attribute?

Reply:
Based on what I see, it looks fine when the page is published, and the spacing is the same as the other pictures.